curriculum/challenges/english/25-front-end-development/lecture-working-with-strings-in-javascript/673263e80dd43da7df3ae565.md

```diff
@@ -86,35 +86,34 @@ Remember, you can pass a second argument to the method to control where the sear
 
 ## --text--
 
-What type of function is `indexOf` in JavaScript?
+What type of method is `indexOf` in JavaScript?
 
 ## --answers--
 
 A standalone function.
 
 ### --feedback--
 
-Methods are functions that belong to specific objects in JavaScript.
+Consider the type of object that deals with text in JavaScript.
 
 ---
 
-A method of the `Array` object.
+A method of the `Date` object.
 
 ### --feedback--
-
-Methods are functions that belong to specific objects in JavaScript.
+Consider the type of object that deals with text in JavaScript.
```

Comment:
```suggestion

Consider the type of object that deals with text in JavaScript.
```
You have accidentally removed a line here, after that I would consider this good to go